packages/selenium/src/AxeBuilder.cs

```diff
@@ -210,34 +247,195 @@ public AxeResult Analyze()
         }
 
         /// <summary>
-        /// Runs axe via scan.js at a specific context, which will be passed as-is to Selenium for scan.js to interpret, and
-        /// parses/handles the scan.js output per the current builder options.
+        /// Runs axe via legacyScan.js at a specific context, which will be passed as-is to Selenium for legacyScan.js to interpret, and
+        /// parses/handles the legacyScan.js output per the current builder options.
         /// </summary>
-        /// <param name="rawContextArg">The value to pass as-is to scan.js to use as the axe.run "context" argument</param>
+        /// <param name="rawContextArg">The value to pass as-is to legacyScan.js to use as the axe.run "context" argument</param>
         private AxeResult AnalyzeRawContext(object rawContextArg)
         {
-            _webDriver.Inject(_AxeBuilderOptions.ScriptProvider, runOptions);
+            ConfigureAxe();
+
+            var runPartialExists = (bool)_webDriver.ExecuteScript(EmbeddedResourceProvider.ReadEmbeddedFile("runPartialExists.js"));
 
 #pragma warning disable CS0618 // Intentionally falling back to publicly deprecated property for backcompat
```

Comment:
These `#pragma warning` lines should move down to `SerializedRunOptions()` to prevent the warnings being found on L438